numpy/polynomial/__init__.py

```diff
@@ -20,6 +20,54 @@
 from .hermite_e import HermiteE
 from .laguerre import Laguerre
 
+def set_default_printstyle(style):
```

Comment:
PEP 8: Two blank lines before `def`.
```suggestion

def set_default_printstyle(style):
```